### PR TITLE
Use a 1 week duration for custom payload builds

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
/hold

still discussing the impact of this with @bradmwilliams and @petr-muller

File: prow.go [hunk 1]
```diff
@@ -368,7 +368,8 @@ func (m *jobManager) newJob(job *Job) (string, error) {
 	pj.Annotations["ci-chat-bot.openshift.io/expires"] = strconv.Itoa(int(m.maxAge.Seconds() + launchDeadline.Seconds()))
 	prow.OverrideJobEnvVar(&pj.Spec, "CLUSTER_DURATION", strconv.Itoa(int(m.maxAge.Seconds())))
 	if job.Mode == "build" {
-		prow.SetJobEnvVar(&pj.Spec, "PRESERVE_DURATION", "12h")
+		// keep the built payload images around for a week
+		prow.SetJobEnvVar(&pj.Spec, "PRESERVE_DURATION", "168h")
```
Comment (petr-muller) [inline prow.go:372]: We just need to make sure this gets propagated to both `--delete-after` (hard TTL) and `--delete-when-idle` (soft TTL)

Comment (bparees) [reply prow.go:372]: at present it does not:

```
# import the initial release, if any
UNRESOLVED_CONFIG=$INITIAL ARTIFACTS=$(ARTIFACTS)/initial ci-operator \
  --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson \
  --image-mirror-push-secret=/tmp/push-auth \
  --gcs-upload-secret=/secrets/gcs/service-account.json \
  --namespace=$(NAMESPACE) \
  --delete-when-idle=$(PRESERVE_DURATION) \
  "${targets[@]}"
```

Comment (bparees) [reply prow.go:372]: cluster-bot doesn't appear to ever pass `--delete-after`, does ci-operator have a default for it?

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**